docs/index.md

```diff
@@ -223,17 +224,17 @@ $ ./node_modules/.bin/mocha mocha.test.js
 
 ## Assertions
 
-Mocha allows you to use any assertion library you wish.  In the above example, we're using Node.js' built-in [assert](https://nodejs.org/api/assert.html) module--but generally, if it throws an `Error`, it will work! This means you can use libraries such as:
+Mocha allows you to use any assertion library you wish. In the above example, we're using Node.js' built-in [assert](https://nodejs.org/api/assert.html) module--but generally, if it throws an `Error`, it will work! This means you can use libraries such as:
```

Comment:
...if it throws an `AssertionError`,

Reply:
it’s not necessary to throw an `AssertionError` from a failed assertion.

Reply:
Perhaps, but then diffs may not occur. All the packages listed fire `AssertionError`...

Reply:
Mocha generates its own diffs unless the reporter tells Mocha not do to that...